### PR TITLE
fix(docs): Update getStartedLink

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/content/whats-new/2020/09/windows-logs-supported.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 title: Windows logs supported
 summary: Support for Windows logs with New Relic's infrastructure agent
 releaseDate: '2020-09-09'
-getStartedLink: 'https://docs.newrelic.com/docs/integrations/host-integrations/host-integrations-list/windows-event-log-integration'
+getStartedLink: 'https://docs.newrelic.com/docs/infrastructure/host-integrations/host-integrations-list/windows-services-integration/'
 ---
 
 Customers with Windows can now send all their logs to New Relic using the infrastructure agent. The feature includes new filters to select exactly which log types (application, security, or system) and messages you are interested in, all the way down to the `EventID`.
```
Comment (ally-sassman) [inline src/content/whats-new/2020/09/windows-logs-supported.md:8]:
```suggestion
```

**Review by nbaenam — COMMENTED** [review]
Hi @ally-sassman, should we update a what's new post from 2020?

File: src/content/whats-new/2020/09/windows-logs-supported.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 title: Windows logs supported
 summary: Support for Windows logs with New Relic's infrastructure agent
 releaseDate: '2020-09-09'
-getStartedLink: 'https://docs.newrelic.com/docs/integrations/host-integrations/host-integrations-list/windows-event-log-integration'
+getStartedLink: 'https://docs.newrelic.com/docs/infrastructure/host-integrations/host-integrations-list/windows-services-integration/'
```
Comment (ally-sassman) [inline src/content/whats-new/2020/09/windows-logs-supported.md:5]:
```suggestion
getStartedLink: 'https://docs.newrelic.com/docs/integrations/host-integrations/host-integrations-list/windows-event-log-integration'
```

**Comment by ally-sassman** [comment]
@nbaenam Good call. I ended up just adding the old link as a redirect on the up-to-date page.

**Pull request merged**